test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -188,6 +188,8 @@ def test_pytorch_mpi_gpu(pytorch_training, ec2_connection, gpu_only, py3_only, e
     """
     Tests mpi backend
     """
+    if 'huggingface' in pytorch_training and 'trcomp' in pytorch_training:
```

Comment:
Should we mention the framework version as well?

Reply:
we don't support mpi with trcomp